rocketpy/utilities.py

```diff
@@ -194,13 +196,243 @@ def du(z, u):
         interpolation="linear",
     )
 
-    if seeGraphs:
+    if see_graphs:
         altitudeFunction()
         velocityFunction()
 
     return altitudeFunction, velocityFunction, final_sol
 
 
+def fin_flutter_analysis(
+    fin_thickness, shear_modulus, flight, see_prints=True, see_graphs=True
+):
+    """Calculate and plot the Fin Flutter velocity using the pressure profile
+    provided by the selected atmospheric model. It considers the Flutter Boundary
+    Equation that published in NACA Technical Paper 4197.
+    These results are only estimates of a real problem and may not be useful for
+    fins made from non-isotropic materials.
+    Currently, this function works if only a single set of fins is added, otherwise
+    it will use the last set of fins added to the rocket.
+
+    Parameters
+    ----------
+    fin_thickness : float
+        The fin thickness, in meters
+    shear_modulus : float
+        Shear Modulus of fins' material, must be given in Pascal
+    flight : rocketpy.Flight
+        Flight object containing the rocket's flight data
+    see_prints : boolean, optional
+        True if you want to see the prints, False otherwise.
+    see_graphs : boolean, optional
+        True if you want to see the graphs, False otherwise. If False, the
+        function will return the vectors containing the data for the graphs.
+
+    Return
+    ------
+    None
+    """
+
+    # First, we need identify if there is at least a fin set in the rocket
+    for aero_surface, _ in flight.rocket.aerodynamicSurfaces:
+        if isinstance(aero_surface, TrapezoidalFins):

```

Comment:
Very good question. Let me think if we can adapt the equations somehow.
I am still not sure

Reply:
Yes, just confirming here, the current implementation only works for trapezoidal fins. 
It could be adjusted to evaluate the Bending and Torsion modulus for different shapes, but that's something that we don't need to worry right now.